components/eamxx/src/physics/shoc/eamxx_shoc_process_interface.cpp

```diff
@@ -499,17 +506,16 @@ void SHOCMacrophysics::check_flux_state_consistency(const double dt)
 {
   using PC = scream::physics::Constants<Real>;
   const Real gravit = PC::gravit;
-  const Real qmin   = 1e-12; // minimum permitted constituent concentration (kg/kg)
```

Comment:
@bogensch @AaronDonahue @PeterCaldwell this lower bound was taken from EAM setup (iirc EAM won't run if this bount is set to 0.0). Do we need a certain nontrivial bound in scream or can we always assume lower acceptable bounds for tracers are zero. In this case it is only for vapor. If a certain bound is needed, Conrad will put it in physics constants.

Reply:
maybe it is a question for @brhillman instead, since in EAM the comment mentions radiation
```
    real(r8),intent(in)    :: qminc  ! minimum value of mass mixing ratio (kg/kg)
                                     ! normally 0., except water 1.E-12, for radiation.
```